README.md

```diff
@@ -102,18 +102,44 @@ val paymentFlowHandler = object : PaymentFlowHandler {
 }
 ```
 
-### Step 2: Create a configuration object for your Payment form
+### Step 2: Prepare your object responsible for the Frames configuration
 ```kotlin
+/*
+  PrefillData is purely optional. However, if you provide these details for your user you can
```

Comment:
Change to:

Providing `PrefillData` is optional. However, using this can increase the speed of the checkout experience for your users by pre-filling non-sensitive information.